wiki/osu!_rankings_history/en.md

```diff
@@ -0,0 +1,392 @@
+| ![][flag_DE] [WhiteCat](https://osu.ppy.sh/users/4504101) | 2019-10-06 | *2021-04-08* | [\[1\]](https://web.archive.org/web/20191006200709/https://osu.ppy.sh/rankings/osu/performance "Internet Archive — osu!") [\[2\]](https://ameobea.me/osutrack/user/whitecat/ "osu!track") [\[3\]](https://web.archive.org/web/20210119102542/https://old.reddit.com/r/osugame/comments/de8duf/whitecat_is_now_1_global_on_osustandard/ "Internet Archive — Reddit") [\[4\]](https://web.archive.org/web/20210203170702if_/https://www.youtube.com/watch?v=Bx4R7lovF-0 "Internet Archive — YouTube") |
+
+## 2020
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_DE] [WhiteCat](https://osu.ppy.sh/users/4504101) | *2019-10-06* | *2021-04-08* | [\[1\]](https://web.archive.org/web/20191006200709/https://osu.ppy.sh/rankings/osu/performance "Internet Archive — osu!") [\[2\]](https://ameobea.me/osutrack/user/whitecat/ "osu!track") [\[3\]](https://web.archive.org/web/20210119102542/https://old.reddit.com/r/osugame/comments/de8duf/whitecat_is_now_1_global_on_osustandard/ "Internet Archive — Reddit") [\[4\]](https://web.archive.org/web/20210203170702if_/https://www.youtube.com/watch?v=Bx4R7lovF-0 "Internet Archive — YouTube") |
+
+## 2021
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_DE] [WhiteCat](https://osu.ppy.sh/users/4504101) | *2019-10-06* | 2021-04-08 | [\[1\]](https://web.archive.org/web/20191006200709/https://osu.ppy.sh/rankings/osu/performance "Internet Archive — osu!") [\[2\]](https://ameobea.me/osutrack/user/whitecat/ "osu!track") [\[3\]](https://web.archive.org/web/20210119102542/https://old.reddit.com/r/osugame/comments/de8duf/whitecat_is_now_1_global_on_osustandard/ "Internet Archive — Reddit") [\[4\]](https://web.archive.org/web/20210203170702if_/https://www.youtube.com/watch?v=Bx4R7lovF-0 "Internet Archive — YouTube") |
+| ![][flag_AU] [rushia1 (*mrekk*)](https://osu.ppy.sh/users/7562902) | 2021-04-08 | Present | [\[1\]](https://osu.ppy.sh/rankings/osu/performance) [\[2\]](https://osu.ppy.sh/users/7562902) [\[3\]](https://ameobea.me/osutrack/user/mrekk/ "osu!track") [\[4\]](https://old.reddit.com/r/osugame/comments/mmkaag/mrekk_is_now_1_surpassing_whitecat/ "Reddit") [\[5\]](https://osu.ppy.sh/scores/osu/3584256449") [\[6\]](https://www.reddit.com/r/osugame/comments/mmkajm/mrekk_colorsslash_colors_power_ni_omakasero/ "Reddit") [\[7\]](https://www.youtube.com/watch?v=xQLVNqfqaOE "YouTube") |
+
+## Number of reigns
```

Comment:
this section, as well as "Longest combined reign length" and "Longest individual reigns", needs a date stamp (time stamp, even) -- the real numbers change eventually and it'll take some digging to figure out which date they belonged to

Reply:
I can't entirely agree that the Number of Reigns and Longest combined reign length needs date stamps adding to them as I think they would make the tables harder to read at a glance. Still, I can see how including them for the Longest individual reigns table would be beneficial.

Reply:
I don't mean a time stamp for every entry, just for the whole table ("captured on 2021-mm-dd")

Reply:
Oh, I see. Yeah, I can see how that would be useful. 

~~Though a more elegant solution would be for wiki articles to have when they were last modified included in them by default~~